examples/plot_function_counter.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+"""
+======================================
+Identifying function names in a script
+======================================
+
+Calls Sphinx-Gallery identify names function to figure out which
+functions are called in the script and to which module do they belong.
+"""
+
+# Code source: Óscar Nájera
+# License: BSD 3 clause
+
+import matplotlib.pyplot as plt
+import sphinxgallery.backreferences as spback
+
+
+filename = spback.__file__.replace('.pyc', '.py')
+file_code = open(filename, 'r').read()
```

Comment:
Thinking about it you could even simplify the example by just passing a string to spback.identify_names. Something like:

```python
python_code = """
from os import path as opath

def (filename):
  return opath.exists(filename)
"""
python_code = """
from os import path as opath

def test(filename):
  return opath.exists(filename)
"""

names = spback.identify_names(python_code)
```